creator-node/scripts/run-tests.sh

```diff
@@ -12,6 +12,26 @@ export dbUrl="postgres://postgres:postgres@localhost:$PG_PORT/audius_creator_nod
 export storagePath='./test_file_storage'
 export logLevel='info'
 
+export COMPOSE_PROJECT_NAME="cn1" 
+export CREATOR_NODE_DB_HOST_PORT=4432 
+export CREATOR_NODE_REDIS_HOST_PORT=4379 
+export CREATOR_NODE_HOST_PORT=4000 
+
+if [ "$1" == "restart" ]; then
```

Comment:
@vicky-g this is very much an ad-hoc implementation, if you have any suggestions on how to better do this would love to hear them!

Reply:
discussed via slack -- might be good idea to add an check here to see if the creator node is already running. if it is, use that. else, bring up and tear down for test! 

also could we have a comment here describing the purpose of this if block addition?

Reply:
also the arg `restart` is a little confusing. perhaps `standalone-creator-node`?